crates/solidity/testing/sanctuary/src/tests.rs

```diff
@@ -123,3 +138,24 @@ fn extract_compiler_version(compiler: &str) -> Option<Version> {
 
     Some(version)
 }
+
+fn uses_exotic_parser_bug(file: &Path) -> bool {
+    static CONTRACTS_WITH_EXOTIC_PARSER_BUGS: &[&str] = &[
+        // 0.4.24: // Accepts malformed `* /` in multi-line comments:
```

Comment:
I’ve always considered that “if it has real-world (mainnet) usage, we should support it”, as others trying to parse contracts will also run into it. But not sure if supporting `/* abc * /` is even possible with our current infra due to backtracking. WDYT?

Reply:
Update: We discussed that we will not support the first two cases - these are obvious bugs that will require us to modify the parser and/or DSL and it may have a performance implication.

Reply:
Updated the list to only include the first two cases in https://github.com/NomicFoundation/slang/pull/896/commits/14d44feff9e29833edee0e200d1c786d21da4176.